packages/components/src/components/action-pad/action-pad.browser.e2e.tsx

```diff
@@ -155,7 +154,7 @@ describe("calcite-action-pad", () => {
         </calcite-action-pad>,
       );
 
-      const [action1, action2] = el.querySelectorAll("calcite-action");
+      const [action1, action2, action4, action5] = el.querySelectorAll("calcite-action");
```

Comment:
Array destructuring assignment follows the same order as the input, so this is actually storing actions 1 – 4.